setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='pypif',
-      version='1.3.0',
+      version='2.0.0',
```

Comment:
Why is this a major version bump? Is it because there is functionality that has been removed?

Reply:
Yep.  Its not backwards compatible.